docs/_interface/pruning.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Pruning
+sort key: F
+---
+
+A snap-sync'd Geth node currently requires more than 650 GB of disk space to store the 
+historic blockchain data. With default cache size the database grows by about 14 GB/week. 
+This means that Geth users will rapidly run out of space on 1TB hard drives. To solve this 
+problem without needing to purchase additional hardware, Geth can be pruned. Pruning is the 
+process of erasing older data to save disk space. Since Geth `v1.10`, users have been able 
+to trigger a snapshot offline prune to bring the total storage back down to the original 
+~650 GB in about 4-5 hours. This has to be done periodically to keep the total disk storage 
+within the bounds of the local hardware (e.g. every month or so for a 1TB disk).
+
+To prune a Geth node at least 40 GB of free disk space is recommended. This means pruning 
+cannot be used to save a hard drive that has been completely filled. A good rule of thumb 
+is to prune before the node fills ~80% of the available disk space.
+
+## Pruning rules
+
+1) Do not try to prune an archive node. Archive nodes need to maintain ALL historic data by 
+   definition.
+2) Ensure there is at least 40 GB of storage space still available on the disk that will be 
+   pruned. Failures have been reported with ~25GB of free space.
+3) Geth is at least `v1.10` ideally > `v1.10.3`
+4) Geth is fully sync'd
+5) Geth has finished creating a snapshot that is at least 128 blocks old. This is true when 
+   "state snapshot generation" is no longer reported in the logs.
+
+With these rules satisfied, Geth's database can be pruned.
+
+## How pruning works
+
+Pruning uses snapshots of the state database as an acceleration structure to determine which 
+nodes in the state trie can be kept and which ones are stale and can be discarded. Geth 
+rebuilds the state trie from a stored snapshot that is older than 128 blocks, discarding any 
```

Comment:
Geth identifies the target state trie based on a stored snapshot layer which has at least 128 block confirmations on top(for surviving reorgs)